apps/pay/app/layout.tsx

```diff
@@ -2,10 +2,10 @@ import type { Metadata } from "next"
 import { Inter_Tight } from "next/font/google"
 
 // eslint-disable-next-line import/no-unassigned-import
-import "./globals.css"
+import "bootstrap/dist/css/bootstrap.css"
 
 // eslint-disable-next-line import/no-unassigned-import
```

Comment:
```suggestion
// eslint-disable-next-line import/no-unassigned-import
```

is this still necessary?

Reply:
yes, these comments work for next line only. We can add comment to ignore in entire file, but not sure if that is better approach.